src/Cafe/IOSU/legacy/iosu_fpd.cpp

```diff
@@ -165,9 +165,28 @@ namespace iosu
 				g_fpd.myPresence.isOnline = 1;
 				g_fpd.myPresence.gameKey.titleId = CafeSystem::GetForegroundTitleId();
 				g_fpd.myPresence.gameKey.ukn = CafeSystem::GetForegroundTitleVersion();
+
+				// Resolve potential domain to IP address
+				struct addrinfo hints = {0}, *addrs;
+				hints.ai_family = AF_INET;
+
+				const int status = getaddrinfo(nexTokenResult.nexToken.host, NULL, &hints, &addrs);
+				if (status != 0) {
+#ifdef _WIN32
```

Comment:
```suggestion
#if BOOST_OS_WINDOWS
```
Checking for `_WIN32` is fine, but we use the boost OS constants everywhere else so it'd be nice if this also follows that.